### PR TITLE
Backmerge Commcare 2.53.1

### PR DESCRIPTION
**Pull request by avazirna**
## Summary
This PR back-merges changes from CommCare 2.53.1 into the `master` branch.

### CODE REVIEW

File: app/AndroidManifest.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
           xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
           android:versionCode="106"
-          android:versionName="2.53">
+          android:versionName="2.53.1">
```
Comment (ctsims) [inline app/AndroidManifest.xml:6]: I think the versionName tags shouldn't change on backmerges, right?

**Review by ctsims — COMMENTED** [review]
one question / feedback on general merge approach

File: app/AndroidManifest.xml (hunk 1 above)
Comment (avazirna) [reply app/AndroidManifest.xml:6]: Correct, let me fix this

**Comment by avazirna** [comment]
@damagatchi retest this please

**Review by ctsims — APPROVED** [review]

**Comment by avazirna** [comment]
@damagatchi retest this please

**Pull request merged**